### PR TITLE
Update the pipeline to be 1ES Compliant

### PR DESCRIPTION
**Pull request by wandeg**
Used the pipeline conversion tool. Tested by running the pipeline on the 1esconversion branch.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoftgraph/kibali/pull/80)

### CODE REVIEW

**Review by andrueastman — DISMISSED** [review]
Looks good to me. 

@wandeg I'd like @MIchaelMainer to take a second look before merging if that's okay.

**Comment by wandeg** [comment]
> Looks good to me.
> 
> @wandeg I'd like @MIchaelMainer to take a second look before merging if that's okay.

Sounds good. Thanks

File: .azure-pipelines/ci-build.yml [hunk 1]
```diff
@@ -2,287 +2,272 @@
...
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-2022
```
Comment (MIchaelMainer) [inline .azure-pipelines/ci-build.yml:28]: Let's future proof this with `image: windows-latest`

**Review by MIchaelMainer — APPROVED** [review]
Approval conditional based on updating the target image name to `windows-latest`

**Review by andrueastman — APPROVED** [review]

**Pull request merged**